### PR TITLE
solution 1

### PR DESCRIPTION
**Pull request by pr1syazhniy**
[DEMO LINK](https://pr1syazhniy.github.io/layout_catalog/)
[TEST REPORT LINK](https://pr1syazhniy.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    cursor: pointer;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    padding: 23px 0;
+    margin-left: 20px;
+    position: relative;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #060b35;
```
Comment (slavanosatskyi) [inline src/styles/blocks/header.scss:58]: move reused colors into separate variable

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
+$background-color: #fff;
+$primary-text-color: #060b35;
+$secondary-text-color: #616070;
+
+html {
```
Comment (slavanosatskyi) [inline src/styles/blocks/card.scss:5]: you already defined it in another file. Delete it or move to main.scss

File: src/styles/blocks/page.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.page {
+  font-family: Roboto, sans-serif;
```
Comment (slavanosatskyi) [inline src/styles/blocks/page.scss:2]: you have already defined it in html selector, so you can remove it from here

File: src/styles/blocks/page.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+  &__body {
+    margin: 0;
```
Comment (slavanosatskyi) [inline src/styles/blocks/page.scss:5]: your margins are reseted in normalize.scss. And I'm not sure where do you use this class. The same goes to page.scss. Seems to be redundant file

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
looks nice, but code has to be polished

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: content-box;
```
[the remark this answers is not in the document]
Comment (OleziO) [reply src/styles/blocks/card.scss:6]: Declare it only in `*` selector and remove it from all other places.

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,105 @@
+$background-color: #fff;
```
Comment (OleziO) [inline src/styles/blocks/card.scss:1]: Declare variables in a separate file.

**Review by OleziO — CHANGES_REQUESTED** [review]
Make links more meaningful.

**Review by slavanosatskyi — APPROVED** [review]

**Pull request closed without merging**